bin/clean-tsv-metadata

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+import pandas as pd
+import argparse
+
+def clean_metadata_file(file_name, n_cols, header):
+    """
+    1. Read the metadata TSV file
+    2. Sort alphabetically and make sure n columns (n-1 tabs) each row
+    3. Write out (overwrite) metadata file
+    """
+    col_names = list(pd.read_csv(file_name, sep="\t", header=None, nrows=1).iloc[0]) if header else list(range(n_cols))
+    metadata = pd.read_csv(file_name,
```

Comment:
If you want this to work with an annotations file, you'll want to add `comment = '#'` as an argument.

Reply:
I just tried doing this and some effects of it are:
1. it removes all of the commented lines from the beginning of the file - this is ok since we could parse and re-add them with grep
2. it removes comments on the end of lines, e.g. 
```
Argentina/PAIS-A0003/2020      EPI_ISL_430795  division        Buenos Aires # previously Ciudad Autonoma de Buenos Aires
```
. This I don't immediately see a way around, but there could be one!

The way that the script works now, i.e. without `comment = '#'` passed to pd.read_csv, it seems to do reasonably well with:
1. fully #-commented lines; they still end up at the top (albeit sorted alphabetically). 
2. lines with comments on the end; they maintain their informative comments. 

Looking at b1f143a, is there anything in particular that it does that we don't want?

Reply:
Ah yes, you're right. My apologies for recommending something without testing it myself. Please ignore that recommendation for now.

Tangentially related to annotations comments, I see that the first row that is a comment in an annotations TSV gets wrapped in double quotation marks (and extra double quotation marks appear around previously quoted strings). Here are my steps to reproduce:
```
head -n 5 source-data/gisaid_annotations.tsv > source-data/test_annotations.tsv 
tail -n 5 source-data/gisaid_annotations.tsv >> source-data/test_annotations.tsv 
cp source-data/test_annotations.tsv source-data/copy_test_annotations.tsv
./bin/clean-tsv-metadata --metadata source-data/copy_test_annotations.tsv --n-cols 4
```
I'm not sure what could possibly be causing this bug.

Reply:
Should be fixed by ab18071!